airbyte-integrations/bases/base-normalization/dbt-project-template/macros/clean_tmp_tables.sql

```diff
@@ -0,0 +1,10 @@
+-- macro to clean Redshift normalization tmp tables
+{% macro redshift__clean_tmp_tables(schemas) %}
+    {%- for tmp_schema in schemas -%}
+        {% do log("\tDROPING SCHEMA " ~ tmp_schema, info=True) %}
+        {%- set drop_query -%}
+            drop schema if exists {{ tmp_schema }} cascade;
```

Comment:
Is it possible to add some `where clauses` as a safeguard and avoid dropping things too violently?

ie the schema to be dropped should validate certain criteria:
- been created in a certain date range
- match a regex pattern
- created by a certain user
- etc

Reply:
Specifically for the Redshift, normalization schemas are created per each test individually with unique suffix like `_fhsfjh` this allows us to track individual tests run from different places at the same time to be unique per requester.
There is no need for `where clause` here because of the uniqueness of the schemas created for the tests. We drop only those what were created and registered in `sources.yml`.